doc/sphinx/language/gallina-extensions.rst

```diff
@@ -85,7 +85,7 @@ To build an object of type :token:`ident`, one should provide the constructor
 
   .. productionlist::
     record_term : {| [`field_def` ; … ; `field_def`] |}
-    field_def : name [binders] := `record_term`
+    field_def : `name` [`binders`] := `record_term`
```

Comment:
I am not aware of `name` being a defined token, so it should probably be `ident` instead.
As for the left hand-side of the `:=` it should probably be `term` and not `record_term`.

Reply:
`name` is `ident` or `_`, so in this case `ident` is indeed right; thanks! You're also right about `record_term`.